fullstack-course3-module3-master/triresults/app/models/entrant.rb

```diff
@@ -15,6 +15,7 @@ class Entrant
 
   embeds_one :race, class_name: 'RaceRef'
 
+  embeds_one :racer, as: :parent, class_name: 'RacerInfo', autobuild: true
```

Comment:
you know that you don´t need to specify the class name if you use the default naming, in this case:
`embeds_one :racer_info, as: :parent, autobuild: true`
mongoid will resolve from `:racer_info` that the class is `RacerInfo`

Reply:
Is a good practice to name the embedded document in the same way as the class unless there is a very good reason to not doing it